app/models/concerns/has_role.rb

```diff
@@ -1,7 +1,7 @@
 # frozen_string_literal: true
 module HasRole
   def role
-    Role.find(role_id)
+    Role.where(id: role_id).first
```

Comment:
This looks like it was intended to fail hard.  However the role lookup occurs once the user/project role already has a committed bad role_id.  This would soft_fail it so it would still allow for logging of the bad id commit, and would also prevent the user reports from failing to generate if a user/project role has a bad role_id.

I can revert this back and add some logic in the presenter to check before calling role if needed.